app/Http/Requests/LoginUserRequest.php

```diff
@@ -1,30 +1,53 @@
-<?php
+<?php  
```

Comment:
```suggestion
<?php
```
not that importatnt
but dont use post spaces please !